include/seqan3/io/sequence/sequence_file_format.hpp

```diff
@@ -36,14 +36,21 @@
 
 #pragma once
 
-#include <seqan3/alphabet/nucleotide/dna4_sequence.hpp>
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
 #include <seqan3/io/sequence/sequence_file_in.hpp>
 #include <string>
 #include <fstream>
 
 namespace seqan3
 {
-
+/*!\interface seqan3::sequence_file_format_concept <>
+ * \brief The generic concept for sequence file formats.
+ * \ingroup io
+ *
+ * The requirements for this concept are read and write functions and associated file extension(s).
+ * Types that satisfy this concept are shown as "implementing this interface".
+ */
```

Comment:
You might want to inlcude the following part: (copied from h-2's PR #91)
```
 * \par Concepts and doxygen
 * The requirements for this concept are given as related functions and metafunctions.
 * Types that satisfy this concept are shown as "implementing this interface".
```

Also in his PR, h-2 actually commented on the functions that are required using the tag `\fn`. Take a look at the PR :)